theano/gpuarray/elemwise.py

```diff
@@ -75,10 +75,10 @@ def make_node(self, *inputs):
             pass
         try:
             support_code = self.scalar_op.c_support_code()
-            if (support_code.strip() != "#define THEANO_MACRO_MOD(x,y) (x % y)" and
-                    support_code.strip() != ""):
+            #if (support_code.strip() != "#define THEANO_MACRO_MOD(x,y) (x % y)" and
```

Comment:
@abergeron what would be a good fix for this? Raise an error if "struct" is in the support_code? This is needed for the psi scalar op.